### ISSUE TITLE
Error: The getter 'typeArgumentTypes'

### ISSUE THREAD

**Issue by MahdiPishguy** [simolus3/moor#113]
why i get this error:

```dart
[SEVERE] Failed to snapshot build script .dart_tool/build/entrypoint/build.dart.
This is likely caused by a misconfigured builder definition.
[SEVERE] file:///E:/Software/flutter/.pub-cache/hosted/pub.dartlang.org/moor_generator-1.7.0/lib/src/parser/column_parser.dart:158:38: Error: The getter 'typeArgumentTypes' isn't defined for t
he class 'MethodInvocation'. - 'MethodInvocation' is from 'package:analyzer/dart/ast/ast.dart' ('file:///E:/Software/flutter/.pub-cache/hosted/pub.dartlang.org/analyzer-0.36.3/lib/dart/ast/ast
.dart').Try correcting the name to the name of an existing getter, or defining a getter or field named 'typeArgumentTypes'.          final type = remainingExpr.typeArgumentTypes.single;
                              ^^^^^^^^^^^^^^^^^
pub finished with exit code 78
```

**Comment by simolus3** [comment]
Yikes, I didn't know they only added that getter in the analyzer version 0.37. If you add
```yaml
dev_dependencies:
  analyzer: '>=0.37.0'
```
to your pubspec, does that solve the problem? 

Moor wrongly declares compatibility with older analyzer versions, it shouldn't. I'll release a proper fix for that soon.

**Comment by MahdiPishguy** [comment]
@simolus3 it can solve my problem

**Comment by simolus3** [comment]
FYI, I've just released moor 1.7.1 which fixes this problem. You can remove the `analyzer` dev dependency if you include
```yaml
dev_dependencies:
  moor_generator: ^1.7.1
```
